### PR TITLE
[enh] reduce the number of http outgoing connections.

### PR DESCRIPTION
**Pull request by dalf**
Engines modified by this pull request : 
- google autocompleter
- deviantart (including thumbnail). Without this PR, the https request redirects to the http website. 
- digg for thumbnails
- google image for thumbnails

Engines that still use http : 
- gigablast
- bing image for thumbnails
- 1x
- dbpedia autocompleter

### CODE REVIEW

File: searx/engines/www1x.py [hunk 1]
```diff
@@ -19,8 +19,8 @@
 categories = ['images']
 paging = False
 
-# search-url
-base_url = 'http://1x.com'
+# search-url, no HTTPS
```
Comment (pointhi) [inline searx/engines/www1x.py]: I think the comment is wrong, because this engine would use HTTPS after merging that pull request.

Comment (dalf) [inline searx/engines/www1x.py]: Oups sorry... 
that's not the comment but the code which is wrong : 1x.com doesn't respond on https request (there is a https://secure.1x.com with an invalid certificate)

File: searx/engines/www1x.py [hunk 2]
```diff
@@ -27,6 +27,7 @@
 # do search-request
 def request(query, params):
     params['url'] = search_url.format(query=urlencode({'q': query}))
+    print params['url']
```
Comment (Cqoicebordel) [inline searx/engines/www1x.py]: You should delete this print

**Comment by asciimoo** [comment]
Thanks!

**Pull request merged**